src/components/poster/PosterListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { usePosterListItemQuery } from "./PosterListItem.generated";
+import { css } from "emotion";
+import { Link } from "../core";
+import { AgendaTalksListItemSpeakers } from "../agenda/AgendaTalksListItem";
+import { VSpace } from "../layout";
+export const PosterListItem = ({ posterId }: { posterId: string }) => {
+  const { data, error, loading } = usePosterListItemQuery({
+    variables: { id: posterId },
+  });
+
+  if (error) throw error;
+  if (loading) return <p>Loading...</p>;
+  if (!data?.poster) return <p>Couldn't load this poster...</p>;
+
+  const { title, abstract, speakers } = data.poster;
+  const pdflink = String(data.poster.pdflink);
+
+  return (
+    <div>
+      <Link href={pdflink} as={pdflink}>
+        <h4
+          className={css`
+            font-weight: bold;
+          `}
+        >
+          {title}
+        </h4>
+      </Link>
```

Comment:
This should actually just be an `<a />` tag. You only need the `<Link />` for routes to other pages (this is so the NextJS framework can do smart code-splitting and pre-loading). Also, it's typically good practice to put links inside heading elements instead of the other way around.

```suggestion
      <h4
        className={css`
          font-weight: bold;
        `}
        >
          <a href={pdflink} target="_blank" rel="noopener noreferrer">
            {title}
          </a>
        </Link>
```